src/daos-serialize/daos-serialize.c

```diff
@@ -147,17 +147,17 @@ int main(int argc, char** argv)
     int len = strlen(argpaths[0]); 
 
     int tmp_rc;
-    tmp_rc = daos_parse_path(argpaths[0], len, &daos_args->src_pool_uuid,
-                         &daos_args->src_cont_uuid);
-    if (tmp_rc != 0 || !daos_uuid_valid(daos_args->src_pool_uuid) || !daos_uuid_valid(daos_args->src_cont_uuid)) {
+    tmp_rc = daos_parse_path(argpaths[0], len, &daos_args->src_pool,
+                         &daos_args->src_cont);
+    if (tmp_rc != 0) {
```

Comment:
Don't we still need to make sure we got a source cont? I think duns_resolve_path fails if it can't parse a valid pool, but it succeeds if there is a pool but no container, right?